spec/controllers/campaigns_controller_spec.rb

```diff
@@ -153,9 +153,9 @@
 
     it 'removes the given organizer from the campaign if the current user is a campaign organizer' do
       create(:campaigns_user, user_id: user.id, campaign_id: campaign.id,
-                                                role: CampaignsUsers::Roles::ORGANIZER_ROLE)
+                              role: CampaignsUsers::Roles::ORGANIZER_ROLE)
       allow(controller).to receive(:current_user).and_return(user)
-      put :remove_organizer, params: { slug: campaign.slug, id: user2.id }
+      put :remove_organizer, params: { slug: campaign.slug, id: organizer.id }
```

Comment:
I'm so stupid >:( It originally went off of the CampaignsUsers id, but then I realized in the view we have the User and not the CampaignsUsers, so to make it easier I adjusted the `remove_organizer` action to go off of user ID.